src/dotnet/OpenAPI.Validator.Tests/Resource/OpenAPI/Validation/positive/clean-complex-spec.json

```diff
@@ -75,9 +73,7 @@
           "pets"
         ],
         "x-ms-examples": {
-          "The pet we get": {
-
-          }
+          "The pet we get": {}
```

Comment:
nit: Ideally json keys should not have spaces as such, consider renaming this to `thePetWeGet`